docs/operational_guide/replication_between_clusters.md

```diff
@@ -0,0 +1,73 @@
+# Replication between clusters (beta)
+
+## Overview
+
+M3DB clusters can be configured to passively replicate data from other clusters. This feature is most commonly used when operators wish to run two (or more) regional clusters that function independently, but that passively replicate data from each other in an eventually consistent manner.
+
+The cross-cluster replication feature is built on-top of the [background repairs](repairs.md) feature. As a result, it has all the same caveats and limitations. Speifically, it does not currently work with clusters that use M3DB's indexing feature and the replication delay between two clusters will be at least (block size + bufferPast) for each namespace.
```

Comment:
nit: "on-top" shouldn't be a hyphenate; also consider "leverages" or "utilizes", whichever feels best to you